dev/tests/api-functional/testsuite/Magento/GraphQl/Catalog/ProductViewTest.php

```diff
@@ -918,7 +916,9 @@ private function assertEavAttributes($product, $actualResponse)
                 'expected_value' => $expectedAttribute ? $expectedAttribute->getValue() : null
             ];
         }
-
+        if ($assertionMap[4]['expected_value'] == "US") {
```

Comment:
Please, keep the interpreter performance in mind 

```suggestion
        if ($assertionMap[4]['expected_value'] === 'US') {
```

Reply:
**Done**